force-app/main/default/lwc/serviceTypeIcon/__tests__/serviceTypeIcon.test.js

```diff
@@ -0,0 +1,71 @@
+import { createElement } from "lwc";
+import ServiceTypeIcon from "c/serviceTypeIcon";
+
+jest.mock("c/customIcon");
+describe("c-service-type-icon", () => {
+  const flushPromises = () => new Promise(process.nextTick);
+
+  afterEach(() => {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+  });
+
+  it("Test if showCostCalculator is true", async () => {
+    const element = createElement("c-service-type-icon", {
+      is: ServiceTypeIcon
+    });
+    element.claimtype = "Calculator";
+
+    document.body.appendChild(element);
+    await flushPromises();
+    const showCostCalculator = element.shadowRoot.querySelector(
+      'svg[data-id="showCostCalculator"]'
```

Comment:
you cannot name an element showCostCalculator, the data-id is not a flag but rather an identifier
a more appropiate name would be just costCalculatorImage or even just costCalculator
the same applies for all data-id tags